arch/risc-v/src/rv32im/riscv_vfork.c

```diff
@@ -41,9 +41,9 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-#ifndef CONFIG_STACK_ALIGNMENT
-#  define CONFIG_STACK_ALIGNMENT 4
-#endif
+/* RISC-V requires a 16-byte stack alignment. */
+
+#define STACK_ALIGNMENT     16
```

Comment:
remove? nobody reference it.

Reply:
Thanks, I've removed the unused definition.